src/app/manager-dashboard/reports/myplanet-table.component.ts

```diff
@@ -39,6 +42,17 @@ export class MyPlanetTableComponent implements OnInit, OnChanges, AfterViewInit
 
   ngOnChanges() {
     this.myPlanets.data = this.data;
+    if (this.usage !== undefined) {
+      let total;
+      this.usage.map(data => {
```

Comment:
This is modifying the `data.max.totalUsedTime` variable as a side effect.  This should be avoided.

`Array.map` returns an array of the same size with values based on the function provided.  You should return in the function `{ ...data, max: { ...data.max, totalUsedTime: total } }`